kitsune/sumo/static/sumo/scss/form-wizard.scss

```diff
@@ -0,0 +1,45 @@
+:root,
+:host {
+  --color-progress: var(--color-blue-06);
+}
+
+:host {
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  box-shadow: 0px 2px 6px rgba(58, 57, 68, 0.2);
+}
+
+.form-wizard-content {
+  padding-inline: 32px;
+  padding-block-start: 24px;
+}
+
+h2 {
+  margin: 0;
+  font-size: 1.5rem;
+  line-height: 1;
+  color: var(--color-heading);
+}
+
+ul {
+  margin-inline-end: 48px;
+}
+
+section {
+  display: flex;
+}
+
+progress {
+  width: 100%;
+  height: 8px;
```

Comment:
If we change this to `max-height: 8px;`, then we can use `flex: 1` to flex to the total width. I think we generally prefer using flex rules rather than percentages when we can use them.